### DISCUSSION TITLE
thresh value to use in pyart.retrieve.est_rain_rate_zkdp

### DISCUSSION THREAD

**Discussion by johnnypeste** [ARM-DOE/pyart#1676] [General]
Hi,

Marshall Palmer is good for small rain rates.
kdp is good for large rair ates

zkdp is implemented in:

pyart.retrieve.est_rain_rate_zkdp

where it uses M-P below thresh and kdp above thresh

Does anybody have a good guess about the thresh value to use?

**Comment by scollis** [comment]
This is really a science question not a Py-ART question.Please ask at:https://openradar.discourse.group/ From: johnnypeste ***@***.***>Date: Tuesday, October 29, 2024 at 13:22To: ARM-DOE/pyart ***@***.***>Cc: Subscribed ***@***.***>Subject: [ARM-DOE/pyart] thresh value to use in pyart.retrieve.est_rain_rate_zkdp (Discussion #1676)Hi,Marshall Palmer is good for small rain rates.kdp is good for large rair ateszkdp is implemented in:pyart.retrieve.est_rain_rate_zkdpwhere it uses M-P below thresh and kdp above threshDoes anybody have a good guess about the thresh value to use?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
